table.go

```diff
@@ -133,13 +133,13 @@ func transformFields(m map[string]interface{}) {
 //   VALUES ('cfd66ccc-d857-4e90-b1e5-df98a3d40cd6', 'johndoe')
 //
 // Gotcha: primkey must be first
-func insertStatement(keySpaceName, cfName string, fieldNames []string, opts Options) string {
+func insertStatement(keySpaceName, cfName string, fields map[string]interface{}, opts Options) (string, []interface{}) {
+	fieldNames, vals := keyValues(fields)
+
 	placeHolders := make([]string, len(fieldNames))
-	for i := 0; i < len(fieldNames); i++ {
```

Comment:
We were iterating over `fieldNames` twice here. I combined the two iterations into one 🐭